plugins/Highway_Lanes.py

```diff
@@ -107,6 +111,9 @@ def way(self, data, tags, nds):
                                 err.append({"class": 31606, "subclass": 0 + stablehash64(tl + '|' + t + '|' + str(i)), "text": T_("Unknown turn lanes value \"{0}\"", t)})
                             if (t == "merge_to_left" and i == 0) or (t == "merge_to_right" and i == len(ttt) - 1):
                                 err.append({"class": 31600, "subclass": 1 + stablehash64(tl + '|' + t + '|' + str(i))})
+                            elif (t != tt and t[0:9] == "merge_to_"):
```

Comment:
Could a false positive with multiple `"merge_to_` ?

Reply:
Also, check the other is a turn lane.

Reply:
Thanks for the review!

> Could a false positive with multiple "merge_to_ ?

You mean a single lane to have `merge_to_right` and `merge_to_left`? I.e. `left|merge_to_right;merge_to_left|right`? (I think duplicate values on a single lane are invalid, right? Like `through|merge_to_left;merge_to_left`)

> Also, check the other is a turn lane.

I don't understand what you mean, sorry. 
If I'm not mistaken, the only other options of `t` here in this part of the loop would be [slight_/sharp_]right/left, through, reverse (or none, but that's invalid here I think, as `none` implies there's no lane marking).

Reply:
> I don't understand what you mean, sorry.

Or do you mean I should check vs. `unknown` being False?

Reply:
Ok I reread the code. I was confused by the check into the loop.

You can do it before the loop. Do the split before the loop and check in the list if there is a `merge_to_` and other values.

Reply:
Ah I just read this comment too late and went for another approach with (I think) the same result; 
I don't know if you'd like me to change it?

I think your approach would be (dummy code):
- take `set(tt.split(";"))` out of the loop (let's call it X here)
- `if ((len(X) > 2 and ("merge_to_right" in X or "merge_to_left" in X)) or ("merge_to_right" in X and not "merge_to_left" in X) or (not "merge_to_right" in X and "merge_to_left" in X)):` -> error

(If so, I'll also move the other merge_to_* check out of the loop)

Reply:
Yes, please, move out of the loop.

Reply:
👍 Done!
Better name suggestions for settt ("set of tt") are welcome ;)

Reply:
The `t` in the `subclass` calculation must be replaced by something else now, as `t` is a loop variable, but I'm not entirely sure what to do with it. Can I just put `tt` there, or could that result in non-unique subclasses?

Reply:
I think, in the vast majority of cases, t == tt, so the effect should be small